{"kind":"symbol","identifier":{"interfaceLanguage":"swift","url":"doc:\/\/OpenAPIKit\/documentation\/OpenAPIKit\/ValidationError\/reason"},"hierarchy":{"paths":[["doc:\/\/OpenAPIKit\/documentation\/OpenAPIKit","doc:\/\/OpenAPIKit\/documentation\/OpenAPIKit\/ValidationError"]]},"primaryContentSections":[{"kind":"declarations","declarations":[{"tokens":[{"kind":"keyword","text":"let"},{"kind":"text","text":" "},{"kind":"identifier","text":"reason"},{"kind":"text","text":": "},{"preciseIdentifier":"s:SS","kind":"typeIdentifier","text":"String"}],"platforms":["Linux"],"languages":["swift"]}]}],"variants":[{"traits":[{"interfaceLanguage":"swift"}],"paths":["\/documentation\/openapikit\/validationerror\/reason"]}],"abstract":[{"type":"text","text":"The reason for the validation failure."}],"metadata":{"modules":[{"name":"OpenAPIKit"}],"role":"symbol","roleHeading":"Instance Property","fragments":[{"kind":"keyword","text":"let"},{"kind":"text","text":" "},{"kind":"identifier","text":"reason"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"String","preciseIdentifier":"s:SS"}],"externalID":"s:10OpenAPIKit15ValidationErrorV6reasonSSvp","title":"reason","symbolKind":"property"},"schemaVersion":{"major":0,"minor":3,"patch":0},"sections":[],"references":{"doc://OpenAPIKit/documentation/OpenAPIKit":{"role":"collection","kind":"symbol","abstract":[],"url":"\/documentation\/openapikit","identifier":"doc:\/\/OpenAPIKit\/documentation\/OpenAPIKit","type":"topic","title":"OpenAPIKit"},"doc://OpenAPIKit/documentation/OpenAPIKit/ValidationError/reason":{"fragments":[{"kind":"keyword","text":"let"},{"kind":"text","text":" "},{"text":"reason","kind":"identifier"},{"text":": ","kind":"text"},{"text":"String","kind":"typeIdentifier","preciseIdentifier":"s:SS"}],"kind":"symbol","identifier":"doc:\/\/OpenAPIKit\/documentation\/OpenAPIKit\/ValidationError\/reason","url":"\/documentation\/openapikit\/validationerror\/reason","title":"reason","type":"topic","role":"symbol","abstract":[{"type":"text","text":"The reason for the validation failure."}]},"doc://OpenAPIKit/documentation/OpenAPIKit/ValidationError":{"role":"symbol","kind":"symbol","navigatorTitle":[{"kind":"identifier","text":"ValidationError"}],"fragments":[{"kind":"keyword","text":"struct"},{"kind":"text","text":" "},{"kind":"identifier","text":"ValidationError"}],"abstract":[{"type":"text","text":"Validation errors are just a textual reason for validation failure and"},{"type":"text","text":" "},{"text":"a coding path where the validation error occurred.","type":"text"}],"url":"\/documentation\/openapikit\/validationerror","identifier":"doc:\/\/OpenAPIKit\/documentation\/OpenAPIKit\/ValidationError","type":"topic","title":"ValidationError"}}}